mover-rsync-tls/server.sh

```diff
@@ -8,12 +8,19 @@ RSYNCD_CONF=/tmp/rsyncd.conf
 STUNNEL_CONF=/tmp/stunnel.conf
 STUNNEL_PID_FILE=/tmp/stunnel.pid
 PSK_FILE=/keys/psk.txt
-STUNNEL_LISTEN_PORT=8000
 RSYNC_LOG=/tmp/rsyncd.log
+IPV6_ENABLED=$(cat /sys/module/ipv6/parameters/disable)
```

Comment:
This is pretty minor, but could we call this variable IPV6_DISABLED?  I think the intention would be clearer (particularly when comparing to 0 later).